open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -323,22 +330,219 @@ DEFINE
+ VALIDLOANNAMETYPES        = NUMBER ARRAY(99)
+ VALIDSHARENAMETYPES       = NUMBER ARRAY(99)
+ VALIDNAMETYPESCHR         = CHARACTER
+ VALIDNAMETYPES            = NUMBER ARRAY(99)
+
+ BNODELIMITER              = NUMBER
+ BNODELIMNONE              = 0
+ BNODELIMQUOTE             = 34
+ BNODELIMCR                = 13
+ BNODELIMLF                = 10
+ BNODELIMCOMMA             = 44
+ BNODELIMTAB               = 09
+ BNODELIMVB                = 124
+ BNODELIMCARET             = 94
+ BNODELIMPIPE              = 124
+ BNODELLIMSEMICOLON        = 59
+
+ ERRORCODEDESCR            = CHARACTER ARRAY(1,19)
+ ECDEFAULT                 = 0
+ ECCUSTOM                  = 1
+
+ NAMETYPESMAX              = 53
+
+ COPYNAMELIST              = CHARACTER ARRAY(36,20)
+ COPYNAMELISTERROR         = CHARACTER ARRAY(20)
+ CNLERRORCHR               = CHARACTER
+ COPYNAMELISTCOUNT         = NUMBER
+ COPYNAMECOUNTMAX          = NUMBER
+ CNLSSN                    = 0
+ CNLLAST                   = 1
+ CNLFIRST                  = 2
+ CNLSTREET                 = 3
+ CNLEXTRAADDRESS           = 4
+ CNLCITY                   = 5
+ CNLSTATE                  = 6
+ CNLZIPCODE                = 7
+ CNLLOC                    = 8
+ CNLMBRADDRNUMLINK         = 9
+ CNLMEMBERNUMLINK          = 10
+ CNLTYPE                   = 11
+ CNLSOURCE                 = 12
+ CNLBIRTHDATE              = 13
+ CNLLOCATOR                = 14
+ CNLSHORTNAME              = 15
+ CNLLONGNAME               = 16
+ CNLADDRESSTYPE            = 17
+ CNLSUBTYPE                = 18
+ CNLNAMEFORMAT             = 19
+ CNLSSNTYPE                = 20
+ CNLSSNOVERRIDE            = 21
+ CNLSSNCERTIFICATION       = 22
+ CNLFOREIGNTIN             = 23
+ CNLUSPERSONFLAG           = 24
+ CNLTITLE                  = 25
+ CNLMIDDLE                 = 26
+ CNLSUFFIX                 = 27
+ CNLPHONETYPE              = 28
+ CNLHOMEPHONE              = 29
+ CNLCARRIERROUTE           = 30
+ CNLCOUNTRYCODE            = 31
+ CNLCOUNTRY                = 32
+ CNLRESTRICT               = 33
+ CNLLASTADDRVERIFDATE      = 34
+ CNLSEX                    = 35
+ CNLDUPLICATED             = 36
+
+ CNTYPE                    = NUMBER
+ NAMETYPESHARE             = 0
+ NAMETYPELOAN              = 1
+
+ MAXERRORCODE              = 11
 END
 
 SETUP
- DEBUGMODE=DEBUGMODEOFF
+ BNODEBUGMODE=BNODEBUGMODEOFF
+ TRUEFALSECHR(FALSE)="false"
+ TRUEFALSECHR(TRUE)="true"
+ SACSHAREGROUPSMAX=19
+
+ NAMEMERGEACTIVE=GETDATANUMBER(GETPARAMNAMEMERGEENABLE)
+
+ FOR BNOLOCALLOOP=1 TO BNOINDENTMAX
+  DO
+   BNOINDENT(BNOLOCALLOOP)=REPEATCHR(" ",BNOLOCALLOOP)
+  END
 
- IF DEBUGMODE=DEBUGMODEON THEN
+
+ IF BNODEBUGMODE=BNODEBUGMODEON THEN
   DATAFILETYPE="LETTER"
  ELSE
   DATAFILETYPE="DATA"
 
  Q=CTRLCHR(34)
- PROGRAMVERSION="1.0.9"
- LASTMODDATE='05/12/21'
- LASTMODTIME="16:30 PT"
- PROGRAMUPDATENOTE1="Corrected JSon output when no shares have and available balance"
- PROGRAMUPDATENOTE2="Enhanced Program info to JSON Output"   
+ QQ=CTRLCHR(34)
```

Comment:
I also am not sure if we need two CTRLCHR(34) variables, unless sometimes QQ makes the code more readable?